### DISCUSSION TITLE
Does anyone is having issues getting the email with the authentication key to login?

### DISCUSSION THREAD

**Discussion by jjchiw** [codemagic-ci-cd/codemagic-docs#2130] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Other - please mention in your question

### Steps to reproduce

1. Go to Login Page
2. Type the email
3. Click Login
4. Email doesn't arrive :(

### Expected results

Email should be received so I can login with the authentication key

### Actual results

No email received

**Comment by himesh-cm** [comment, marked as answer]
Hi @jjchiw, do you have your team ID? If not, connect via the contact [form](https://codemagic.io/contact/) and share your email so we can assist.

**Reply by jjchiw** [reply]
Hi, sorry for the late response.

A colleague is giving the follow up via the contact form.

Thanks
